src/main/java/com/smartbear/readyapi/client/assertions/HttpStatusCodeAssertionBuilder.java

```diff
@@ -5,9 +5,13 @@
 import java.util.List;
 
 public interface HttpStatusCodeAssertionBuilder extends AssertionBuilder<Assertion> {
-    HttpStatusCodeAssertionBuilder addStatusCode(int statusCode);
+    HttpStatusCodeAssertionBuilder withStatusCode(int statusCode);
```

Comment:
sorry to be a pain - but let's keep the old signatures around with deprecated annotations... since we have some users and these are probably one of the more common assertions